test/Test/BaseDAO/Token.hs

```diff
@@ -7,89 +7,38 @@ module Test.BaseDAO.Token
 
 import Universum
 
-import Lorentz
 import Morley.Nettest
 import Morley.Nettest.Tasty (nettestScenario)
 import Test.Tasty (TestTree, testGroup)
-import Util.Named
 
+import qualified BaseDAO.ShareTest.Token as Share
 import qualified Lorentz.Contracts.BaseDAO.Types as DAO
-import qualified Lorentz.Contracts.Spec.FA2Interface as FA2
 import Test.Common
 
 {-# ANN module ("HLint: ignore Reduce duplication" :: Text) #-}
 
 test_BaseDAO_Token :: TestTree
 test_BaseDAO_Token = testGroup "BaseDAO non-FA2 token tests:"
   [ testGroup "Admin:"
-    [ nettestScenario "can burn tokens from any accounts" adminBurnScenario
-    , nettestScenario "can mint tokens to any accounts" adminMintScenario
+    [ nettestScenario "can burn tokens from any accounts"
+        $ uncapsNettest $ Share.adminBurnScenario True
+          $ originateTrivialDaoWithBalance
+            (\o1 _ ->
+                [ ((o1, DAO.unfrozenTokenId), 10)
+                , ((o1, DAO.frozenTokenId), 10)
+                ]
+            )
+    , nettestScenario "can mint tokens to any accounts"
+        $ uncapsNettest $ Share.adminMintScenario True
+          $ originateTrivialDaoWithBalance
+            (\o1 _ ->
+                [ ((o1, DAO.unfrozenTokenId), 0)
+                , ((o1, DAO.frozenTokenId), 0)
+                ]
+            )
     ]
   , testGroup "Other:"
-    [ nettestScenario "can call transfer tokens entrypoint" transferContractTokensScenario
```

Comment:
same comment on dividing test groups as the one just above, I think the top-most test group is fine now.